opentelemetry-semantic-conventions/src/opentelemetry/semconv/Client_attributes.py

```diff
@@ -0,0 +1,36 @@
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-lines
+
+from enum import Enum
+
+class ClientAttributes:
+
+    
+    CLIENT_ADDRESS = "client.address"
```

Comment:
Q:

1. `ClientAttributes.CLIENT_ADDRESS`
2. `ClientAttributes.ADDRESS`

Probably 1 to allow 

```python

from ... ClientAttributes import CLIENT_ADDRESS
from ... serverAttributes import SERVER_ADDRESS
```

Reply:
`ClientAttributes.CLIENT_ADDRESS` seems to go with the pattern of having the namespace be in the variable name so I'd prefer that one.